web/settings/integration_test.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/cozy/cozy-stack/model/instance/lifecycle"
```

Comment:
I would prefer to not rename this file to `integration_tests.go`. Currently, what we are calling integration tests are in https://github.com/cozy/cozy-stack/tree/master/tests/integration, and I would like to avoid the confusion between the two.